pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -257,96 +258,61 @@ class StateVectorKokkos final
     }
 
     /**
-     * @brief Apply a single qubit operator to the state vector using a matrix
+     * @brief Apply a multi qubit operator to the state vector using a matrix
      *
      * @param matrix Kokkos gate matrix in the device space
      * @param wires Wires to apply gate to.
      * @param inverse Indicates whether to use adjoint of gate.
      */
-    void applySingleQubitOp(const KokkosVector &matrix,
```

Comment:
Just for my recollection, the `applySingleQubitOp` function is now replaced by something else?

Reply:
Everything is under `applyMultiQubitOp`, and `singleQubitOpFunctor`, `twoQubitOpFunctor` are replaced respectively with `apply1QubitOpFunctor`, `apply2QubitOpFunctor` to have a more consistent naming (we also have `apply3QubitOpFunctor`, `apply4QubitOpFunctor`, etc.)